api/src/main/resources/messages.properties

```diff
@@ -1,4 +1,8 @@
 ${project.parent.artifactId}.title=Bahmni Appointment Scheduling
 
 bahmni.appointment.teleConsultation.patientEmailNotificationSubject=Teleconsultation appointment confirmed
-bahmni.appointment.teleConsultation.patientEmailNotificationTemplate=Hi {0}, Your tele-consultation appointment with {1} for {2} has been scheduled! Please use this link {3}. If you have any questions, please reach out to administration for assistance. See you soon!
\ No newline at end of file
+bahmni.appointment.teleConsultation.patientEmailNotificationTemplate=Hi {0}, Your tele-consultation appointment with {1} for {2} has been scheduled! Please use this link {3}. If you have any questions, please reach out to administration for assistance. See you soon!
+
+bahmni.appointment.adhocTeleConsultation.patientEmailNotificationSubject=AdhocTeleconsultation started
+bahmni.appointment.adhocTeleConsultation.patientEmailNotificationTemplate=Hi {0}, {1} has started an adhoc tele-consultation with you! Please use this link {3}. If you have any questions, please reach out to administration for assistance. See you soon!

```

Comment:
Same comment. No need for the word "adhoc".

Reply:
Updated